src/Engine/GraphLayout/GraphLayout.csproj

```diff
@@ -1,63 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What's the advantage of using the sdk style project - I see there's multitargeting, any other?

Reply:
less code...and will be easier to move to newer versions of dotnet

Reply:
And I do plan to use multitargeting to have a dotnet6 build as well (at least for the non UI projects)